services/metrics/alarmMetricService.js

```diff
@@ -9,17 +9,14 @@ async function createAlarmLog(userID, timeStart, timeEnd) {
   let alarm = new AlarmMetricModel(null, userID.toString(), timeStart, timeEnd, false);
 
   try {
-    await metricDB("alarmlog").insert({
+    alarmLogID = await metricDB("alarmlog").insert({
       userid: alarm.userid,
       alarmstart: alarm.timeStart,
       alarmend: alarm.timeEnd,
       alarmsent: alarm.wasSent
-    }).returning("*").then(res => {
-      console.log(res);
-      alarmLogID = res[0].id;
-    });
+    }).returning("*")

```

Comment:
You can specify to return the id only here to prevent the [0].id call on line 19. Same with the other functions.